--- src/plugins/backends/aqhbci/plugin/Makefile.am.link	2010-08-04 20:52:33.000000000 +0200
+++ src/plugins/backends/aqhbci/plugin/Makefile.am	2010-08-04 20:55:50.000000000 +0200
@@ -71,6 +71,7 @@
 
 testlib_SOURCES=testlib.c
 testlib_LDADD = \
+  libaqhbci.la \
   $(aqhbci_internal_libs) \
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs)
--- src/plugins/backends/aqhbci/plugin/Makefile.in.link	2010-08-19 23:09:02.000000000 +0200
+++ src/plugins/backends/aqhbci/plugin/Makefile.in	2010-08-21 20:38:13.000000000 +0200
@@ -107,7 +107,7 @@
 PROGRAMS = $(noinst_PROGRAMS)
 am_testlib_OBJECTS = testlib.$(OBJEXT)
 testlib_OBJECTS = $(am_testlib_OBJECTS)
-testlib_DEPENDENCIES = $(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
+testlib_DEPENDENCIES = libaqhbci.la $(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1)
 DEFAULT_INCLUDES = -I.@am__isrc@ -I$(top_builddir)
 depcomp = $(SHELL) $(top_srcdir)/depcomp
@@ -493,6 +493,7 @@
 aqhbci_la_LDFLAGS = -no-undefined -module -avoid-version
 testlib_SOURCES = testlib.c
 testlib_LDADD = \
+  libaqhbci.la \
   $(aqhbci_internal_libs) \
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs)
